ensime_shared/protocol_handler.py

```diff
@@ -0,0 +1,249 @@
+import webbrowser
```

Comment:
Minor style suggestion: since file names become modules in Python, `protocol.py` could be nicer here, and a little more general if anything else ends up logically belonging here. Then imports read a little better:

```python
from ensime_shared.protocol import ProtocolHandlerV1
```